packages/language/src/language-server/code-actions/quick-fix-resolve-include.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { URI } from "../../utils/uri";
+import { FileSystemProviderInstance } from "../../workspace/file-system-provider";
+import { UriUtils } from "../../utils/uri";
+import {
+  CodeAction,
+  CodeActionKind,
+  Diagnostic,
+} from "vscode-languageserver-types";
+import { Connection } from "vscode-languageserver";
+
+const CONFIG_FOLDER = ".pliplugin";
+const PROC_GRPS_CONFIG_FILE = "proc_grps.json";
+
+export async function quickFixResolveInclude(
+  diagnostic: Diagnostic,
+  connection: Connection,
+): Promise<CodeAction | undefined> {
+  const unresolvedFile = diagnostic.data?.unresolvedFile;
+  if (!unresolvedFile) return undefined;
+
+  const results = await FileSystemProviderInstance.findFilesByGlob(
+    `**/${unresolvedFile}`,
+  );
```

Comment:
Suggestion: If the unresolved file is something like `test_file`, but the file exists as `test_file.pli` on the file system, it won't be picked up by the glob pattern. You need to specify the list of possible file extensions in the glob as well.